src/DeviceInput/InputDevices/deviceSourceManager.ts

```diff
@@ -25,7 +26,7 @@ export class DeviceSourceManager {
     public readonly onDeviceDisconnectedObservable: Observable<DeviceSource<DeviceType>>;
 
     // Private Members
-    private _deviceSourceManager: InternalDeviceSourceManager;
+    private _deviceSourceManager: Nullable<InternalDeviceSourceManager>;
 
     // Public Functions
```

Comment:
These function lambdas must be readonly.

Reply:
function lambdas have been made readonly at the internalDSM level

Reply:
Update: modified to follow public get/private set pattern